docs/commandline.rst

```diff
@@ -158,6 +158,11 @@ Options
    If given, the results will be converted into XML. Note that not all the
    information is kept. This is specifically targeted to Jenkin's plugin CCM.
 
+.. option:: --xml
```

Comment:
Shouldn't this be `--md`?

Reply:
yep... you are completely right. Fixing 😄